### DISCUSSION TITLE
Upgraded gcc compiler to 14.2 and Apple SDK and now get undefined symbols for architecture arm64

### DISCUSSION THREAD

**Discussion by quant79** [gabime/spdlog#3255] [Q&A]
My error message looks like this:

Undefined symbols for architecture arm64:
  "fmt::v10::vformat[abi:cxx11](fmt::v10::basic_string_view<char>, fmt::v10::basic_format_args<fmt::v10::basic_format_context<fmt::v10::appender, char>>)", referenced from:
      ...
  "spdlog::logger::err_handler_(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char>> const&)", referenced from:
      ...
ld: symbol(s) not found for architecture arm64

**Comment by tt4g** [comment]
All libraries must be rebuilt after upgrading the compiler.
